configure-slic/action.yml

```diff
@@ -0,0 +1,71 @@
+name: Configure slic
+description: Installs and configures slic
+
+inputs:
+  wp_version:
+    required: false
+    default: ''
+    description: WordPress version to be installed
+  airplane_mode:
+    required: false
+    default: 'on'
+    description: Airplane mode, no external connections (on/off)
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout slic
+      uses: actions/checkout@v4
+      with:
+        repository: stellarwp/slic
+        ref: main
```

Comment:
I think this should also be an input that defaults to `main`, but we'll need the ability to test different branches/old versions for debugging.